README.md

```diff
@@ -2,11 +2,25 @@
 
 The Jake and trOY devops-y utility
 
+- [joy v0.3.3](#joy-v033)
```

Comment:
Perhaps we remove the version from here?  I find it gets missed a lot when updating and is a bit redundant with the full changelog below.

Reply:
Agreed